inc/rest-api/class-post-fields.php

```diff
@@ -290,7 +290,7 @@ function( $post_type ) {
 	 * @return object
 	 */
 	private function remove_recursion( $fm ) {
-		if ( 'richtext' === $fm->field_class || 'radio' === $fm->field_class ) {
+		if ( 'richtext' === $fm->field_class || 'radio' === $fm->field_class || 'select' === $fm->field_class ) {
```

Comment:
might benefit from an `in_array()` for readability

```suggestion
		if ( in_array( $fm->field_class, [ 'radio', 'richtext', 'select' ], true ) ) {
```